internal/testrand/rand.go

```diff
@@ -0,0 +1,86 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package testrand provides random generation and flags for testing.
+package testrand
+
+import (
+	"flag"
+
+	"golang.org/x/exp/rand"
+)
+
+var (
+	seedFlag    = flag.Uint64("testrand.seed", 1, "random seed for tests (0=randomize)")
+	extremeFlag = flag.Float64("testrand.extreme", 0, "probability of returning extreme values")
+	nanFlag     = flag.Float64("testrand.nan", 0, "probability of returning nan")
+)
+
+// TB is an interfaces that corresponds to *testing.T and *testing.B
```

Comment:
```suggestion
// TB is an interface that corresponds to *testing.T and *testing.B
```
(also, perhaps it's more correct to say `it corresponds to a subset of`...)